addons-dev/aldryn-addons/aldryn_addons/utils.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+
+
+def boolean_ish(value):
+    if isinstance(value, basestring):
+        value = value.lower()
+    if value in [True, 'true', '1', 'on', 'yes']:
+        return True
+    elif value in [False, 'false', '0', 'off', 'no']:
+        return False
+    return bool(value)
+
+
+def json_from_file(path):
+    try:
+        with open(path) as fobj:
+            return json.load(fobj)
+    except ValueError as e:
+        raise ValueError('{} ({})'.format(e, path))
+
+
+def mkdirs(path):
+    try:
+        os.makedirs(path)
+    except:
```

Comment:
@czpython what would you like to log here? If the path already exists, there is nothing to do and all is fine (no log required). And if there is an exception, it is serious... so aborting completely with an exception should be ok.